### PR TITLE
added fix for issue Footer in user profile page not at bottom of screen when user has less photos. #462

### PR DESCRIPTION
**Pull request by xavier-kong**
Tested when user has more photos than page can fit without scrolling and when user has few photos.
image

### CODE REVIEW

**Comment by vnugent** [comment]
`h-screen` paints the white background only to the end of the screen.  When you scroll down, the rest is in black.

See https://open-tacos-c4rmdmvwz-openbeta-dev.vercel.app/?v=newTags

Screen Shot 2022-08-01 at 3 46 02 PM

**Comment by xavier-kong** [comment]
I am using hard override dark mode on my machine so I did not catch that!

File: src/components/layout.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ interface LayoutProps {
 
 function Layout ({
   contentContainerClass = 'content-fullscreen-tablet',
-  rootContainerClass = 'root-container-default',
+  rootContainerClass = 'root-container-default flex flex-col',
```
Comment (vnugent) [inline src/components/layout.tsx:15]: how about adding `flex flex-col` to `root-container-default` class in 
https://github.com/OpenBeta/open-tacos/blob/user-page-footer/src/styles/defaults.css#L16

**Review by vnugent — CHANGES_REQUESTED** [review]

**Comment by vnugent** [comment]
@xavier-kong can you have a look at this?

**Pull request closed without merging**